test/macs_test.py

```diff
@@ -0,0 +1,28 @@
+import pytest
+from pyDFT import  mac_function
+
+def test_numeral():
+
+    """
+    Test the assigned numeral string is returned as an integer
+    :return:
+    """
+
+    assert mac_function.roman_numeral('I') == 1
```

Comment:
you could test the others too in this function. dont need to change it though really